src/pycapsule.rs

```diff
@@ -147,6 +147,10 @@ impl PyCapsule {
     /// Context itself, is treated much like the value of the capsule, but should likely act as
     /// a place to store any state managment when using the capsule.
     ///
+    /// # Safety
+    ///
+    /// This might dereference the raw pointer `context`, which is unsafe.
```

Comment:
Does it? How?

Reply:
Good question. :sweat_smile: 

I tried to place the unsafe block within the method, but clippy doesn't agree:
```
error: this public function might dereference a raw pointer but is not marked `unsafe`
   --> src/pycapsule.rs:182:72
    |
182 |         let result = unsafe { ffi::PyCapsule_SetContext(self.as_ptr(), context) as u8 };
    |                                                                        ^^^^^^^
    |
    = note: `#[deny(clippy::not_unsafe_ptr_arg_deref)]` on by default
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#not_unsafe_ptr_arg_deref

error: could not compile `pyo3` due to previous error
```

Reply:
I think clippy is pulling that out of its steely behind, so yes, please ignore it. No `unsafe` needed on this side.